src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -136,6 +187,10 @@ public BooleanSupplier getOutsideLimit() {
         }
     }
 
+    public IntakeMap getIntakeMap() {
```

Comment:
Why was this moved?

Reply:
It was at the bottom at the file so I moved it right below the intake map